src/core/Akka.MultiNodeTestRunner.Shared.Tests/MultiNodeTestRunnerDiscovery/DiscoverySpec.cs

```diff
@@ -62,6 +63,7 @@ public void Only_MultiNodeConfig_role_count_used()
 
         private static Dictionary<string, List<NodeTest>> DiscoverSpecs()
         {
+            Environment.SetEnvironmentVariable(MultiNodeFactAttribute.MultiNodeTestEnvironmentName, "1");
```

Comment:
What's this for?

Reply:
"Flag to mark that the current test is being run inside the MNTR" according to @Arkatufus in chat - got it